### DISCUSSION TITLE
How to increase brightness in Apple Thunderbolt Display as second monitor.

### DISCUSSION THREAD

**Discussion by Drremmick** [MonitorControl/MonitorControl#1544] [Q&A]
I have an iMac desktop I connected to Thunderbolt Display. I downloaded the controlmonitor app but cannot increase the brightness anymore. I have the slider but it is still dull compared to the iMac brightness. I read somewhere you can set max brightness at 255 but I can’t change anything but the slider.

**Comment by rmohns** [comment]
I am having the same experience with an older Apple Cinema HD Display (the ~2005 vintage - Aluminum chassis, white plastic sides with touch controls; using Apple's USB-C to DVI-D adapter). Max brightness using Monitor Control is considerably less than I can get to with the controls on the display itself.

As a workaround, I've set the display manually to max brightness, then disabled DDC. It's not ideal, because it runs the backlight at full bore all the time. But it does the job, kind of, until I can figure out what's wrong with DDC control:

<img width="719" alt="Screenshot 2024-05-18 at 1 16 18 PM" src="https://github.com/MonitorControl/MonitorControl/assets/2131133/8b7a941c-9af0-4949-9e2d-62a0b0e785c1">

(Apple's own display protocol isn't displayed as an option, which puzzles me too. The display does appear in the USB device tree.)
